src/test/java/io/appform/dropwizard/sharding/dao/locktest/LockTest.java

```diff
@@ -322,9 +320,9 @@ public void testCreateOrUpdate() throws Exception {
                 .stream()
                 .findFirst()
                 .get();
-        Assert.assertEquals(newChildValue, savedChild.getValue());
-        Assert.assertNotEquals(child.getId(), savedChild.getId());
-        Assert.assertEquals(newParentValue, lookupDao.get(parentId).get().getName());
+        assertEquals(newChildValue, savedChild.getValue());
+        Assertions.assertNotEquals(child.getId(), savedChild.getId());
```

Comment:
Be consistent with static imports and direct calls. Ideally it would be excellent if we can move to static imports everywhere.